spec/requests/foods_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe 'FoodsController', type: :request do
+  describe 'GET /foods/' do
+    before(:all) do
+      user = User.find_by(email: 'wassimchakib@gmail.com')
+      user.skip_confirmation!
+      login_as(user)
+      get foods_path
+    end
+
+    # Get lists of all foods
+    it 'returns a 200 status response' do
+      expect(response).to be_successful
+      expect(response).to have_http_status(200)
+    end
+  end
+
+  describe 'GET /foods/:food_id' do
+    before(:all) do
+      user = User.find_by(email: 'wassimchakib@gmail.com')
+      user.skip_confirmation!
+      login_as(user)
+      get recipe_path(1)
+    end
+
+    # Get Specific Recipe
+    it 'returns a 200 status response' do
+      expect(response).to be_successful
+      expect(response).to have_http_status(200)
```

Comment:
You have some failing tests it would be nice if you corrected this.

![Screenshot 2023-03-17 at 9 03 34 PM](https://user-images.githubusercontent.com/79506105/225995505-ab4d58cb-6be4-47c1-abb1-1042ca200dcd.png)